### PR TITLE
build(slider): Bazelify slider component.

### PR DESCRIPTION
**Pull request by SkaillZ**
### <strong>Pull Request</strong>

Bazelify slider component.

#### Type of PR

Other

### CODE REVIEW

File: libs/barista-components/slider/src/slider.spec.ts [hunk 1]
```diff
@@ -493,8 +493,7 @@ describe('DtSlider', () => {
       } = getElements(fixture);
       const { slider } = testComponent;
 
-      //<any, any> is needed because _updateSlider is private
-      const update = jest.spyOn<any, any>(slider, '_updateSlider');
+      const update = jest.spyOn(slider, '_updateSlider');
```
Comment (SkaillZ) [inline libs/barista-components/slider/src/slider.spec.ts:496]: This lead to errors when trying to build with Bazel

Comment (rowa-audil) [reply libs/barista-components/slider/src/slider.spec.ts:496]: We should fix this before we can merge this

Comment (peter-affenzeller) [inline libs/barista-components/slider/src/slider.spec.ts:496]: @SkaillZ said: This lead to errors when trying to build with Bazel

Comment (peter-affenzeller) [reply libs/barista-components/slider/src/slider.spec.ts:496]: it works this way:
```suggestion
      const update = jest.spyOn(slider as any, '_updateSlider');
```
then you don't have to change the class.

**Review by peter-affenzeller — CHANGES_REQUESTED** [review]

**Review by lukasholzer — APPROVED** [review]

**Review by bardosmisi — DISMISSED** [review]

**Review by rowa-audil — APPROVED** [review]

**Review by peter-affenzeller — APPROVED** [review]

**Review by bardosmisi — APPROVED** [review]

**Review by lukasholzer — APPROVED** [review]

**Pull request merged**